### DISCUSSION TITLE
How to get absolute m6A number in each library?

### DISCUSSION THREAD

**Discussion by gaoyubang** [GoekeLab/xpore#136] [Q&A] [answered]
Dear Ploy N. Pratanwanich:

    Without the pvalue fillter in significance level from z-test of the differential modification rate, I get  so many m6A sites, just like below.
   
![image](https://user-images.githubusercontent.com/26346115/157189761-ecb3aa01-c058-498f-9f93-52e4946170ed.png)

Is this correct ？How can i get the absolute m6A number in each library?

**Comment by jonathangoeke** [comment, marked as answer]
Hi @gaoyubang 
the z-score and p-value can be used to rank differentially modified sites. You can set a threshold to determine the number of differentially modified positions identified by xPore. However, this number is different from the absolute number of modified sites (xPore only returns posistions which are different across samples, you can try [m6Anet](https://github.com/GoekeLab/m6anet) for non differentially modified sites). The number should also not be interpreted as the absolute number of differentially modifies sites, it can only be interpreted in relation to the threshold and otherpossible influences such as sequencing throughput, the samples that are used, etc..
I hope that helps!
Jonathan
